protocol/x/revshare/keeper/revshare_test.go

```diff
@@ -726,6 +726,48 @@ func TestKeeper_GetAllRevShares_Valid(t *testing.T) {
 				affiliatesKeeper *affiliateskeeper.Keeper) {
 			},
 		},
+		{
```

Comment:
From thread, could we also add a unit test with positive maker fees and zero taker fee?

Reply:
good catch!